db/seeds.rb

```diff
@@ -137,6 +137,13 @@
 patient_in_completed_calls.calls.create! status: 'Left voicemail',
                                         created_by: user
 
+# Create insurance and external pledge source keysets
+Config.destroy_all
+Config.create config_key: :insurance,
```

Comment:
It inherits stuff from Mongoid, so it basically counts as an activerecord object with all the trappings (like find, find_or_create, pluck, etc)! Kinda handy!